aim/ext/transport/client.py

```diff
@@ -3,19 +3,27 @@
 from copy import deepcopy
 import grpc
 import threading
+from typing import Tuple
+from collections import defaultdict
 import aim.ext.transport.remote_tracking_pb2 as rpc_messages
 import aim.ext.transport.remote_tracking_pb2_grpc as remote_tracking_pb2_grpc
 
 from aim.ext.transport.message_utils import pack_stream, unpack_stream, raise_exception
 from aim.ext.transport.config import AIM_CLIENT_SSL_CERTIFICATES_FILE
-from aim.ext.task_queue.queue import TaskQueue
+from aim.ext.task_queue.queue import TaskQueueWithRetry
 from aim.storage.treeutils import encode_tree, decode_tree
 
 
+DEFAULT_RETRY_INTERVAL = 0.1  # 100 ms
+DEFAULT_RETRY_COUNT = 1
+
+
 class Client:
     _thread_local = threading.local()
 
-    _queue = TaskQueue('metric_tracking', max_backlog=10_000_000)
+    _queues = defaultdict(lambda: TaskQueueWithRetry(
+        'remote_tracker', max_queue_memory=16 * 1024 * 1024 * 1024, register_at_exit=False,
```

Comment:
16 MB might be too small... lets say, images are tracked; a single batch might have 32 images ~3MB each. 
@mahnerak what do you think a reasonable max queue size (in MBs) should be?

Reply:
default to 1GB change by env. variable

Reply:
done